ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/AbstractBlockParameterMethod.java

```diff
@@ -27,7 +27,7 @@
 
 public abstract class AbstractBlockParameterMethod implements JsonRpcMethod {
 
-  private final Supplier<BlockchainQueries> blockchainQueries;
+  protected final Supplier<BlockchainQueries> blockchainQueries;
```

Comment:
Why widen the scope (encapsulation) instead of using the getter (`getBlockchainQueries`)?

Reply:
For a short while we need access to this. When we add the mainnet functionality this can be set back to private. I have added a TODO for making this private again.